jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Dump.java

```diff
@@ -609,7 +616,13 @@ public static void setDumpOptions(String dumpOptions) throws InvalidDumpOptionEx
 		 * progress and the dump configuration could not be updated.
 		 */
 		synchronized (dumpLock) {
-			setDumpOptionsImpl(dumpOptions);
+			try {
+				setDumpOptionsImpl(dumpOptions);
```

Comment:
Please move the synchronization within the try block.

Reply:
Good plan.  Done in both places.